client/src/app/components/home/home.component.html

```diff
@@ -1 +1,35 @@
-Home page check
\ No newline at end of file
+<div class="home-container">
+	<img class="top left corner" src="../../../assets/flowerbar.png" />
+	<img class="top right corner" src="../../../assets/flowerbar.png" />
+	<img class="bottom left corner" src="../../../assets/flowerbottom.png" />
+	<img class="bottom right corner" src="../../../assets/flowerbottom.png" />
+
+	<div class="content-container">
+		<div class="names">
+			<h1>Lena</h1>
+			<div class="ampersand">
+				<h1>&</h1>
+			</div>
+			<h1>Lucas</h1>
+		</div>
+
+		<div>
+		<div class="dateandlocationinfo">
```

Comment:
Styling complaint >:(

hyphen separate words in class name in accordance with css convention pls

I am lodging this complaint here, but it applies to most of your classes.

Reply:
technically we don't follow full convention anyway but whatever. in accordance with other class names